django_project/geocontext/models/context_service_registry.py

```diff
@@ -17,6 +17,8 @@
     convert_coordinate, parse_gml_geometry, get_bbox)
 from geocontext.models.validators import key_validator
 
```

Comment:
Remove line break here. Also by our coding conventions you should order your imports as:

1) python built in modules first
2) 3rd party imports second
3) modules from your own code base third

See 'Ordering of imports' section here: http://kartoza.com/en/coding-standards/